test/system/src/test/java/io/pravega/test/system/AbstractScaleTests.java

```diff
@@ -33,16 +33,16 @@
 abstract class AbstractScaleTests extends AbstractReadWriteTest {
 
     final static String SCOPE = "testAutoScale" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
-    @Getter(lazy = true)
```

Comment:
There are actually two tests that extend this that need to run their initialize() methods before this gets invoked. So this has semantic impact. Perhaps there should be a setup() method on this class that the subclasses call the populates these values.

Reply:
I have spent some time on this decision.

IMHO the overall sequence will not change significantly. And that 'lazy' stuff is really complex and maybe not needed here.

I tried to understand why it was there but without a real answer.
I can refactor to a setup method if you prefer. Please give me an ack